.circleci/config.yml

```diff
@@ -26,7 +26,7 @@ jobs:
 
             curl -OL https://github.com/google/protobuf/releases/download/v3.7.1/$PROTOC_ZIP
 
-            sudo unzip -o $PROTOC_ZIP -d /usr/local bin/protoc
+            sudo unzip -o $PROTOC_ZIP -d /usr/local bin/protoc include/*
```

Comment:
I don't really like the idea of getting the stdlib via install instructions. I think we'd be better off with a submodule so that we can version control them, and keep it in sync with our rules.

Reply:
Ah you know what, I was thinking that these were the api-common-protos that we were discussing in #84. These ones are probably much less likely to change?

Reply:
This will break for zsh, change `include/*` `'include/*'`.

Reply:
> I don't really like the idea of getting the stdlib via install instructions. I think we'd be better off with a submodule so that we can version control them, and keep it in sync with our rules.

I think you figured this out already, but for clarity: this is the actual install instructions in protoc (unzip the archive you are downloading and look at the readme and it tells you to do precisely this).

Reply:
> This will break for zsh, change `include/*` to `'include/*'`.

Who cares? The image is not using zsh.

Reply:
Ah, fair. I'm using zsh and I copy pasted it :)